composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
> It started on Nov 2014.

About the delay : really really sorry about that, we are trying to change our workflow / reactivity